conf.py

```diff
@@ -143,6 +143,15 @@
 
 html_theme = 'sphinx_rtd_theme'
 
+# Set canonical URL from the Read the Docs Domain
+html_baseurl = os.environ.get("READTHEDOCS_CANONICAL_URL", "")
+
+# Tell Jinja2 templates the build is running on Read the Docs
+if os.environ.get("READTHEDOCS", "") == "True":
```

Comment:
Nit: the default value for the `default` parameter is `None`, no need to tweak it here.

```suggestion
if os.environ.get("READTHEDOCS") == "True":
```

Reply:
It's from their docs, so didn't tweak it. Will leave as is.

(Wonder if there's some type-checking background to passing `""` as default. (without it's typed as `Optional[str]`, with it's `str`).